puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_develop.yaml

```diff
@@ -43,7 +43,4 @@
           only-if-success: false
       - junit:
           results: 'jenkins/reports/unit/*.xml'
-      - release_nightly_build_foreman_rpm:
-          project: packages/foreman/foreman
-      - release_nightly_build_foreman_deb:
-          project: foreman
+      - foreman-develop-release
```

Comment:
Should this pass in the git ref that was just tested (somehow)?

Reply:
Yes....handled here https://github.com/theforeman/foreman-infra/pull/923/files#diff-11e13aa5dc38b0d222ef464601829d4dR8